src/components/Containers.tsx

```diff
@@ -125,6 +139,16 @@ export const SubTitleWrapper = styled.h2`
   margin: 0;
 `;
 
+export const FAQTitleWrapper = styled.h2`
```

Comment:
Maybe a generic `Headline` component could defined instead.
But since it's not needed now, I'm fine with it.

In general, I would always try to define reusable components and apply additional styles on them, when there is a special use case.

Can be refactored later, if there is another `h2` needed though. 👍